src/vue/modules/assets-explorer/api.js

```diff
@@ -0,0 +1,21 @@
+import { ApiCaller } from '@tokend/js-sdk'
+
+let versionPrefix = 'v2'
```

Comment:
It rather excess in the current usage. maybe passing the version as an argument would be better?